test/parallel_api/iterator/iterators.pass.cpp

```diff
@@ -138,12 +138,7 @@ struct test_counting_iterator {
 
         //explicit checks of the counting iterator specific
         // There is a bug in clang when we pass the same arguments in the function
-        if(b[0]!=begin){
-            ::std::cout<<"wrong result with operator[] for an iterator"<<::std::endl;
-            exit(1);
-        }
-        EXPECT_TRUE(*(b + 1) == begin+1, "wrong result with operator+ for an iterator");
-        EXPECT_TRUE(*(b+=1) == begin+1, "wrong result with operator+= for an iterator");
+        test_random_iterator(e);
```

Comment:
Do we need the following line: 
https://github.com/oneapi-src/oneDPL/blob/main/test/parallel_api/iterator/iterators.pass.cpp#L293 ?

Reply:
No, we don't, it seems useless.